gensim/models/lsimodel.py

```diff
@@ -298,8 +298,7 @@ def __init__(self, corpus=None, num_topics=200, id2word=None, chunksize=20000,
             self.id2word = utils.dict_from_corpus(corpus)
             self.num_terms = len(self.id2word)
         else:
-            self.num_terms = (
-                1 + max(self.id2word.keys()) if self.id2word else -1)
+            self.num_terms = 1 + max(self.id2word.keys()) if self.id2word else -1
```

Comment:
Please put brackets around the ternary expression, its reading is a little ambiguous / confusing now.